plugins/woocommerce/includes/widgets/class-wc-widget-layered-nav-filters.php

```diff
@@ -87,8 +87,8 @@ public function widget( $args, $instance ) {
 						}
 
 						$filter_classes = array( 'chosen', 'chosen-' . sanitize_html_class( str_replace( 'pa_', '', $taxonomy ) ), 'chosen-' . sanitize_html_class( str_replace( 'pa_', '', $taxonomy ) . '-' . $term_slug ) );
-
-						echo '<li class="' . esc_attr( implode( ' ', $filter_classes ) ) . '"><a rel="nofollow" aria-label="' . esc_attr__( 'Remove filter', 'woocommerce' ) . '" href="' . esc_url( $link ) . '">' . esc_html( $term->name ) . '</a></li>';
+						$anchor_text = apply_filters( 'woocommerce_widget_layered_nav_term_anchor_text', esc_html( $term->name ), $term, $taxonomy);
```

Comment:
```suggestion
						/**
						 * Allows the attribute term name to be modified before being output.
						 *
						 * @param string $term_name The name of the term.
						 * @param WP_Term $term The term object.
						 * @param string $taxonomy The taxonomy name.
						 *
						 * @since 8.7.0
						 */
						$anchor_text = apply_filters( 'woocommerce_widget_layered_nav_term_anchor_text', $term->name, $term, $taxonomy);
```

In order to meet our linting rules, new filters need to have a doc block. Also, the `esc_html` call should happen _after_ the anchor text is filtered, so it would be better to put it back down on the next line.